src/app/pages/ArticlePage/Byline/index.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { ServiceContextProvider } from '../../../contexts/ServiceContext';
+import Timestamp from '../../../legacy/containers/ArticleTimestamp';
+import { bylineWithNameAndRole } from './fixture';
+import Byline from '.';
+import { withServicesKnob } from '../../../legacy/psammead/psammead-storybook-helpers/src';
+
+interface Service {
+  service: string;
+}
+
+interface Fixture extends Service {
+  children?: JSX.Element;
+}
```

Comment:
I would rename `Service` to `Props` and remove the `interface Fixture` type. You can use Reacts built-in `PropsWithChildren` type and wrap it around your own type, for example `PropsWithChildren<Props>`. This will automatically include type definitions for the `children` prop and let React handle them.

Reply:
😮 , the more I know. Thanks, making the change now